src/OrbitQt/orbithistogram.cpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "orbithistogram.h"
+
+#include <absl/strings/str_cat.h>
+#include <absl/strings/str_format.h>
+#include <qevent.h>
+#include <qnamespace.h>
+#include <qpainter.h>
+#include <qpoint.h>
+#include <qwidget.h>
+
+#include <algorithm>
+#include <cmath>
+#include <cstdint>
+#include <iterator>
+#include <string>
+#include <utility>
+
+#include "DisplayFormats/DisplayFormats.h"
+#include "Statistics/Histogram.h"
+
+constexpr double kRelativeMargin = 0.1;
+
+constexpr uint32_t kTicksNum = 3;
+constexpr int kTickLength = 5;
+
+[[nodiscard]] static int RoundToClosestInt(double x) { return static_cast<int>(std::round(x)); }
```

Comment:
I always get this wrong, but isn't the static redundant for free functions? (Same below)

Reply:
No it would be redundant for const global variables. static for free functions means internal linkage.